.github/workflows/check-package-json.yml

```diff
@@ -1,30 +0,0 @@
-
-# this workflow checks if devDeps match peerDeps
-# and also that the PR increments the version
-# of Maker in comparison of what's in the master branch
-
-name: Check package.json on PRs into master
-
-on:
-  pull_request:
-    branches: [ master ]
-
-jobs:
-  check-package-json:
-    name: Check package.json on PRs into master
-    runs-on: ubuntu-latest
-
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: Fetch master branch
-      run: |
-        git fetch origin master
-    - name: Use Node.js 12.x
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
-    - name: Install NPM dependencies
-      run: npm ci
-    - name: Run package.json checks
-      run: node .github/dep-version-check
```

Comment:
Since this just ran a single Node script, I moved it to the bottom of the PR linting workflow